services/ui-src/src/components/fields/DateField.tsx

```diff
@@ -48,6 +53,7 @@ export const DateField = (props: PageElementProps) => {
         value={displayValue}
         hint={parsedHint}
         errorMessage={errorMessage}
+        {...disabled}
```

Comment:
You're exactly right, and this is a good solution. Here's the commit when CMSDS started allowing passthrough props for this input type: https://github.com/CMSgov/design-system/commit/ef43d2cb1786625402b0120727546b51d4686464#diff-ebbd9086a31139490786c9439153de660e9e93f8e2e7e70fca0cb021f1aef6b3

My only suggestion, which you may take or leave, is to localize the hack spatially. Rather than half up on line 15 and half down here, I would stuff the whole thing right here. It will make for a nice clean diff when we are (someday) able to delete it.

```suggestion
        /*
         * Typescript hack. CmsdsDateField won't recognize
         * passthrough props until @cmsgov/design-system^9.0.0
         * TODO: Unhack, to just `disabled={props.disabled}`
         */
        {...{ disabled: props.disabled } }
```

Reply:
This is brilliant. I was trying to figure out a way to get it all in one place, but didn't think of this so thank you!!!